### PR TITLE
teach BT2 how to interpret hyperlinks

### PR DESCRIPTION
**Pull request by emptyrivers**
also implement the Unknown Spell feature

### CODE REVIEW

File: WeakAurasOptions/Cache.lua [hunk 1]
```diff
@@ -277,19 +277,28 @@ function spellCache.BestKeyMatch(nearkey)
...
 function spellCache.CorrectAuraName(input)
   if (not cache) then
     error("spellCache has not been loaded. Call WeakAuras.spellCache.Load(...) first.")
   end
 
-  local spellId = WeakAuras.SafeToNumber(input);
+  local spellId = WeakAuras.SafeToNumber(input)
+  if type(input) == "string" and input:find("|", nil, true) then
+    spellId = WeakAuras.SafeToNumber(input:match("|Hspell:(%d+)"))
+    print(input:gsub("|", "||"), spellId)
```
Comment (InfusOnWoW) [inline WeakAurasOptions/Cache.lua:290]: there's a print here

Comment (emptyrivers) [reply WeakAurasOptions/Cache.lua:290]: whups

Comment (InfusOnWoW) [reply WeakAurasOptions/Cache.lua:290]: Well... You removed the other prints but not this one.-

Comment (emptyrivers) [reply WeakAurasOptions/Cache.lua:290]: im practicing writing code for details addon

**Review by InfusOnWoW — APPROVED** [review]

**Pull request merged**